ScreenToGif/Windows/Recorder.xaml.cs

```diff
@@ -960,6 +960,10 @@ internal async Task Record()
     /// </summary>
     private async Task Snap()
     {
+        var snapTriggerDelay = GetTriggerDelay();
```

Comment:
The `Snap()` method is triggered 3 ways: Mouse and keyboard events in interaction mode and manually in manual mode. 

Right now, the option to appears just for Interaction mode, but is used for both.
Shouldn't we have the same option also appearing for Manual mode in Options?
Maybe separate the options, one for interaction mode, one for manual mode?

Reply:
Happy to add to other modes, and thought about it.  Also realized the code path is taken by multiple modes but as they were 0 it would have no effect.  Mainly wasn't sure if it as as useful for manual mode but happy to add the option.  Just to confirm that is the desire?

Reply:
Ahhh, yes. The `GetTriggerDelay()` method returns zero for snap mode, I missed that. :)

Yes, it would be nice to have that feature for Manual capture mode as well.
Another settings entry needs to be created and another WrapPanel in Options (same structure and text, just getting a different UserSettings property).
Thank you!